### DISCUSSION TITLE
Caractéristiques chiffrées plus variées

### DISCUSSION THREAD

**Discussion by ElethornGM** [fibreville/minotaure_php#62] [Boite à idées]
Comme discuté dans https://github.com/fibreville/atrpg/discussions/40

Une idée pourrait être de passer de 1 à 20 pour les caractéristiques (corps/esprit par défaut), au lieu de 1 à 6.

Le test pourrait se transformer de cette façon : 
lors d'une épreuve, chaque personnage lance 1D20 et le résultat doit être inférieur ou égal à la caractéristique (donc si j'ai 13 en esprit et fait un test d'esprit, le D20 doit faire 13 ou moins pour réussir).
Les bonus/malus aux épreuves seraient appliqués à la caractéristique : un malus de -3 voudrait dire que mon esprit de 13 passe à 10 pour la durée du test.
Les bonus/malus aux loots s'appliqueraient de la même façon.

ça permettrait deux choses : 
1/ avoir des personnages plus variés
2/ pouvoir accorder des loots moins forts : actuellement un loot de +1 apporte 1/6 donc environ 17% de stat, alors qu'avec 1D20, un +1 apporterait seulement +5%. ça permet d'obtenir plus de flexibilité.

Cela demanderait de revoir les bonus/malus aux épreuves (-2, -1, +1, +2 ne serait pas suffisant, il faudrait aller jusque +5/-5 ou alors avoir un champ libre)

**Comment by uurshin** [comment]
Je recopie ma réponse ici : 

On peut très bien changer le principe et adopter du D20.
Il faudrait alors peut être proposer des difficultés plus granulaires (de 1 à 20), peut être à saisir à l'aide d'un slider (https://refreshless.com/nouislider/pips/) ?

On pourrait également choisir de rendre ça paramétrable par partie (D6, D20, D100) mais c'est plus de boulot de dev.

Du coup comment on transformerait la création de personnage (si on prend en compte qu'on aura à terme + de caracs dont les scores varient entre 1 et 20 ? Je proposerais bien que le total des caracs doive faire un certain score défini par le MJ (toujours avec une valeur par défaut, par exemple 10 * nombre de carac), avec une jauge par carac lors de la création du perso.

C'est un boulot de dev assez conséquent.

**Reply by ElethornGM** [reply]
Avoir soit le choix, soit changer du D6 au D20, me semble une bonne solution. L'outil est bon en tant que tel, mais le D20 apporterait plus de flexibilité (comme dit plus haut).
Après, si c'est beaucoup de boulot de dev, ce ne sera sûrement pas pour tout de suite, mais on peut garder cette idée en backlog...

**Comment by asaurat** [comment]
Le D20 apporte aussi beaucoup de variabilité et de chaos, je suis pas méga fan globalement (je préfère le 3d6 par exemple) mais bon sur un jeu où il y a bcp de monde ça passe ;)

C'est clairement sympa en tout cas d'avoir des plages de valeurs plus larges.
Je bosse sur une modif qui va ajouter une 3e carac potentielle et pour laquelle j'ai cherché des jauges JS/CSS qui seraient sympas lors de la création du perso.. mais j'ai rien trouvé de bien cool. Si vous avez ça, je prend !

**Reply by uurshin** [reply]
Check la librairie au dessus, sinon tu as rangeslider : https://rangeslider.js.org/ qui est très simple et se branche sur les input range natif html.
